### PR TITLE
refine developer page

### PR DESCRIPTION
**Pull request by jiaming0708**
* [this page](https://developer.github.com/apps/using-content-attachments/) style not right, so I add single file to do

* the background color of alert block is not really comfortable, maybe designer can give some advance

*warning*
Screen Shot 2019-04-13 at 22 12 07
*alert*
Screen Shot 2019-04-13 at 22 12 12

> code block is difficult to define, maybe you have other idea
Screen Shot 2019-04-13 at 22 13 42

### CODE REVIEW

File: src/code.scss [hunk 1]
```diff
@@ -70,3 +70,7 @@ tt {
 .pl-mb {
     color: #ccc !important;
 }
+
+pre{
```
Comment (poychang) [inline src/code.scss]: Does use `pre` tag will over kill all style?

**Comment by poychang** [comment]
Developer page should be the way you suggest. Can you separate the developer suggest and alert with 2 PRs?

File: src/code.scss (hunk 1 above)
Comment (jiaming0708) [reply src/code.scss]: mm, I am not sure, maybe let me try it in commentXD

```
// this is test for pre
<pre>
test
</pre>
```

Comment (jiaming0708) [reply src/code.scss]: looks good, not effect other page

**Comment by jiaming0708** [comment]
this PR only leave developer page

**Comment by jiaming0708** [comment]
weird....I can't push over two branches to github...

**Pull request closed without merging**